src/controllers/restaurants.controllers.ts

```diff
@@ -0,0 +1,71 @@
+import { Request, Response } from "express";
+import RestaurantHandler from "../handlers/restaurants.handlers";
+
+export const getAll = async (req: Request, res: Response) => {
+  try {
+    const restaurants = await RestaurantHandler.getAll();
+    res.json(restaurants);
+  } catch (error) {
+    res.status(500).json({ message: "Error: An unexpected error occurred" });
+  }
+};
+
+export const getById = async (req: Request, res: Response) => {
+  try {
+    const restaurantId = req.params.id;
+    const restaurant = await RestaurantHandler.getById(restaurantId);
+    if (!restaurant) {
+      return res.status(404).json({ message: "Restaurant not found" });
+    }
+    res.json(restaurant);
+  } catch (error) {
+    res.status(500).json({ message: "Error: An unexpected error occurred" });
```

Comment:
please do it as well you do the chef handler.  same for all handler and controllers